subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -219,6 +219,15 @@ static void sync_rest_client_data(struct nrf_cloud_rest_context *const rest_ctx,
 	}
 }
 
+static void rest_error_response_log(struct nrf_cloud_rest_context *const rest_ctx)
+{
+	LOG_ERR("REST request was rejected. Response status: %d", rest_ctx->status);
```

Comment:
Please check the flash/RAM usage when `CONFIG_LOG=n` to see if you should do some flagging either within this function or also the function calls.

Reply:
Flagging does not make any difference, flash and RAM consumption are exactly the same in both cases. With `CONFIG_LOG=n`, `rest_error_response_log()` is not present in the map file at all, so it gets optimized out.